dev/devicelab/lib/tasks/perf_tests.dart

```diff
@@ -184,15 +191,18 @@ class BuildTest {
   }
 }
 
+/// Measure application memory usage.
 class MemoryTest {
-  MemoryTest(this.testDirectory, this.testTarget, this.timelineFileName, this.packageName, { this.os });
+  MemoryTest(this.testDirectory, this.packageName, { this.testTarget, this.os });
 
   final String testDirectory;
-  final String testTarget;
-  final String timelineFileName;
   final String packageName;
   final DeviceOperatingSystem os;
 
+  // Path to a flutter driver script that will run after starting the app.
+  // If not specified, then the test will start the app, gather statistics, and then exit.
```

Comment:
nits: use `///` for dartdocs and make the first sentence a standalone paragraph (i.e. separate it from the next sentence by 1 blank line).